Alphabet-to-LB_readme.md

```diff
@@ -20,12 +20,14 @@ This Python script removes punctuation, converts upper to lower-case letters, ha
 
 Currently, this script is only made available as a code sample.  Many points of linguistic complexity are elided, and there is still some room to improve the accuracy of the script when it comes to acronyms and foreign words.  Please don't publish with this script!  You have been warned.
 
-## Config
+## Instructions for Use
 
-For language-specific processing, Alphabet to Linear B takes a configuration file-- a Python script which eliminates many-to-one and one-to-many mappings between script and language, and also outlines the language's orthography and phonology.  Examples have been provided for Turkish and German.
+For language-specific processing, Alphabet to Linear B takes a configuration file-- a Python script which eliminates many-to-one and one-to-many mappings between script and language, and also outlines the language's orthography and phonology.  Alphabet to Linear B also takes a .txt file containing the section of text you would like converted.  Config files and data sets have been provided for Turkish and German.

```

Comment:
Instead of `Alphabet to Linear B`, I would name the script like so: `Alphabet-To-LB.py` (wrap it in single backticks to get the inline code block effect) everywhere you reference it.